### PR TITLE
docs(auto-instrumentations-node): improved readme

### PR DESCRIPTION
**Pull request by CptSchnitz**
## Which problem is this PR solving?

- #497 

## Short description of the changes

- Added a summary of what the package does to the readme.
- Added a list of instrumentations the package loads to the readme.
- Fixed the example so it uses resource instead of serviceName.

### CODE REVIEW

**Review by vmarchaud — APPROVED** [review]

File: metapackages/auto-instrumentations-node/README.md [hunk 1]
```diff
@@ -33,15 +41,28 @@ registerInstrumentations({
     getNodeAutoInstrumentations({
       // load custom configuration for http instrumentation
       "@opentelemetry/instrumentation-http": {
-        applyCustomAttributesOnSpan: (span)=> {
-          span.setAttribute('foo2', 'bar2');
+        applyCustomAttributesOnSpan: (span) => {
+          span.setAttribute("foo2", "bar2");
```
Comment (obecny) [inline metapackages/auto-instrumentations-node/README.md:45]:
```suggestion
          span.setAttribute('foo2', 'bar2');
```

File: metapackages/auto-instrumentations-node/README.md [hunk 2]
```diff
@@ -5,26 +5,34 @@
...
+const { Resource } = require("@opentelemetry/resources");
+const { ResourceAttributes } = require("@opentelemetry/semantic-conventions");
+const { SimpleSpanProcessor } = require("@opentelemetry/tracing");
+const { registerInstrumentations } = require("@opentelemetry/instrumentation");
 
-const provider = new NodeTracerProvider();
+const exporter = new CollectorTraceExporter();
+const provider = new NodeTracerProvider({
+  resource: new Resource({
+    [ResourceAttributes.SERVICE_NAME]: "basic-service",
```
Comment (obecny) [inline metapackages/auto-instrumentations-node/README.md:33]:
```suggestion
    [ResourceAttributes.SERVICE_NAME]: 'basic-service',
```

**Review by obecny — APPROVED** [review]
lgtm, just please replace double quotes with single quotes

**Review by MSNev — APPROVED** [review]

File: metapackages/auto-instrumentations-node/README.md (hunk 1 above)
Comment (CptSchnitz) [reply metapackages/auto-instrumentations-node/README.md]: fixed

File: metapackages/auto-instrumentations-node/README.md (hunk 2 above)
Comment (CptSchnitz) [reply metapackages/auto-instrumentations-node/README.md]: fixed

**Comment by CptSchnitz** [comment]
@obecny looks like the unit test failed because a timeout.

**Pull request merged**